stack/building-pipelines-lsst-io-locally.rst

```diff
@@ -40,12 +40,6 @@ Then set up the `pipelines_lsst_io`_ package with EUPS:
 
    setup -r pipelines_lsst_io
 
-.. warning::
```

Comment:
Does it make sense to say that if you have locally-modified clones setup that you should use `-k` to keep them setup?